content/zh-cn/blog/_posts/2022-08-16-PSP-historical-context/index.md

```diff
@@ -3,6 +3,7 @@ layout: blog
 title: "PodSecurityPolicy：历史背景"
 date: 2022-08-23T15:00:00-0800
 slug: podsecuritypolicy-the-historical-context
+evergreen: true
```

Comment:
```suggestion
evergreen: true
```
Please add the same content behind line 13

Reply:
thx～～